core/app/assets/js/utils/utils.js

```diff
@@ -0,0 +1,9 @@
+export default class Utils {
+	static pxToRem = ( pixels ) => {
```

Comment:
Where is this class used?

Reply:
For spacing in some components.
It needs to be discussed